_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -371,7 +371,7 @@ We can change the appearance of our legend by manually setting different paramet
 * `legend.text`: change the font size
 * `legend.box.background`: add an outline box
 
-```{r modify-legend-plot }
+```{r modify-legend-plot fig.cap="Map of the paths in the study area with large-font and border around the legend."}
```

Comment:
```suggestion
```{r modify-legend-plot, fig.cap="Map of the paths in the study area with large-font and border around the legend."}
```